src/bgw/job.c

```diff
@@ -1054,6 +1056,57 @@ zero_guc(const char *guc_name)
 				(errcode(ERRCODE_INTERNAL_ERROR), errmsg("could not set \"%s\" guc", guc_name)));
 }
 
+extern Oid
+ts_bgw_job_get_funcid(BgwJob *job)
+{
+	ObjectWithArgs *object = makeNode(ObjectWithArgs);
+	object->objname = list_make2(makeString(NameStr(job->fd.proc_schema)),
+								 makeString(NameStr(job->fd.proc_name)));
+	object->objargs = list_make2(SystemTypeName("int4"), SystemTypeName("jsonb"));
+
+	return LookupFuncWithArgs(OBJECT_ROUTINE, object, false);
```

Comment:
This function call will generate an error if the function does not exist, and will cause strange errors as a result inside `ts_bgw_job_entrypoint`. You probably want to either avoid calling this function for jobs that do not have a function defined, or allow this function to fail without an error so that the caller can check this and proceed without raising an error.

Reply:
Changed the `missing_ok=true` so the caller will deal with the `InvalidOid`.